src/Codeception/Step/Meta.php

```diff
@@ -1,27 +1,35 @@
 <?php
+
+declare(strict_types=1);
+
 namespace Codeception\Step;
 
 use Codeception\Lib\ModuleContainer;
 use Codeception\Step as CodeceptionStep;
+use function array_pop;
+use function end;
+use function is_string;
+use function str_replace;
+use function strpos;
 
 class Meta extends CodeceptionStep
 {
-    public function run(ModuleContainer $container = null)
+    public function run(ModuleContainer $container = null): void
     {
     }
 
-    public function setTraceInfo($file, $line)
+    public function setTraceInfo($file, $line): void
     {
         $this->file = $file;
         $this->line = $line;
     }
 
-    public function setPrefix($actor)
+    public function setPrefix($actor): void
     {
         $this->prefix = $actor;
     }
 
-    public function getArgumentsAsString($maxLength = 200)
+    public function getArgumentsAsString($maxLength = 200): string
```

Comment:
int

Reply:
I will make this change in a future PR when I edit the files in the path: `src/Codeception/*.php`